packages/popper/src/use-popper/index.ts

```diff
@@ -233,27 +233,18 @@ export default function(
       }
     }
 
+    const triggerEventsMap: Partial<Record<TriggerType, (keyof GlobalEventHandlers)[]>> = {
+      click: ['onclick'],
+      hover: ['onmouseenter', 'onmouseleave'],
+      focus: ['onfocus', 'onblur'],
+    }
+
```

Comment:
This is great, but I think the value needs to be corrected, since `onMouseEnter` has been changed accordingly. `onMouseenter`, you might need to give

Reply:
The names of `GlobalEventHandlers` seems to be all lowercase.

> https://developer.mozilla.org/zh-CN/docs/Web/API/GlobalEventHandlers/onmouseenter

Do we need to change the `onmouseenter`?

Reply:
This is no DOM API it was prepared for Vue.h() for the virtual DOM, you can check out the file called trigger.ts under the renderer folder

Reply:
Yeah, i add `IEvents` for it. And rename `onmouseenter` to `onMouseenter`.

Reply:
Oops, i forget rebase and push it. Just a moment.